governance/program/src/state/governance.rs

```diff
@@ -16,11 +23,10 @@ pub struct GovernanceConfig {
 
     /// Voting threshold in % required to tip the vote
     /// It's the percentage of tokens out of the entire pool of governance tokens eligible to vote
-    pub vote_threshold: u8,
+    pub vote_threshold_percentage: u8,
 
-    /// Minimum % of tokens for a governance token owner to be able to create a proposal
-    /// It's the percentage of tokens out of the entire pool of governance tokens eligible to vote
-    pub token_threshold_to_create_proposal: u8,
+    /// Minimum number of tokens a governance token owner must possess to be able to create a proposal
+    pub min_tokens_to_create_proposal: u16,
```

Comment:
I think percentage is not the correct solution to the problem.

Ideally we would need some histogram and then say the level of tokens covering let's say 90% token holders is the level required to create a `Proposal`. This way we would be able to say the bottom 10% can't create Proposals, however you would need all token accounts to calculate that level which is not practical. 

The problem with percentage is that you can accidentally deny rights to create Proposals for everybody if you mint more governance tokens.
Let's say we set the threshold to 1% for population of 100 users with 1 token each. It means anybody can create a Proposal because 1 token is required.
However if you mint 100 more tokens to 100 more users you would need 2 tokens to create Proposal which means nobody could create `Proposals` at that point. 

This is why I think the absolute number of tokens to create a Proposal is better solution for now and each community has to decide on that level. It's not a gatekeeper anyway. It's just a way to prevent spamming governance with Proposals.
In future versions we should allow the Governance program to govern its Config and hence the thresholds could be updated via Proposals